scripts/test/helpers/event.js

```diff
@@ -6,7 +6,9 @@ export function triggerEvent(element, type, data = {}) {
 
   for (const key in data) {
     if (data.hasOwnProperty(key)) {
-      event[key] = data[key];
+      Object.defineProperty(event, key, {
+        value: data[key],
+      });
```

Comment:
It was complaining about writing to `event.cancelable` in tests, so forcing overwrites here. Should be fine, because we are in a testing environment